client/app/components/Input/InputDefault.jsx

```diff
@@ -60,7 +70,7 @@ export const InputDefault = (props: Props) => {
   } = props;
   if (!REQUIRES_DEFAULT.includes(type)) return null;
   return (
-    <input
+    <><input
```

Comment:
I'm wondering why `<>` was added.

Reply:
so I was only returning one element. I changed it to a div